### PR TITLE
Fixes #1061 : Changed all S0 to 1.0

### LINKED ISSUE #1061: Inconsistency in specifying S0 values in multi_tensor and single_tensor
As pointed out by @etpeterson in https://github.com/nipy/dipy/pull/1058#discussion_r64775504

> Looks like dipy may be a little conflicted also. Single_tensor uses S0=1 and multi_tensor uses S0=100, though others seem to prefer the 100 area.

In dipy/sims/voxel.py multi_tensor specifies S0=100 while single_tensor specifies S0=1

Should it be consistent over both functions ?

### PR DESCRIPTION
**Pull request by sahmed95**
There were specific places where S0=123. which I changed to 100.

### CODE REVIEW

**Comment by arokem** [comment]
I think we want to leave all the tests as they were. We want to make sure that things work for arbitrary inputs. For example, if a user decides they want to use S0=123 for some reason, it should still work. 

Does that make sense?

**Comment by sahmed95** [comment]
Yes. I think so. That's why I pointed it out as I wasn't sure whether to change the tests. I will keep the tests as they were and update the PR.

Thanks

**Comment by sahmed95** [comment]
@I guess I should break this down into multiple commits ? Makes it easier to review and pinpoint where the S0 values should be changed. 

Or multiple PR's ? @arokem

**Comment by arokem** [comment]
No need for multiple PRs or more commits. This is fine. But there is a test failure that needs to be addressed.

**Comment by sahmed95** [comment]
Ah, @arokem I made by pushing without commiting

**Comment by arokem** [comment]
@sahmed95 : did you mean to close this PR? It was going in the right direction, and I think we should keep going here, unless you have an alternative plan.

**Comment by arokem** [comment]
Oh - cool. I see you reopened this right as I sent my message. All good.

**Comment by sahmed95** [comment]
Sorry, that was accidental. I had rebased with master and started fresh and forgot to commit the changes before a forced push. I have made the changes and tests pass locally. There is one place though which still has S0 = 1.

**Comment by arokem** [comment]
Any chance to revisit this one, and complete it?

**Comment by sahmed95** [comment]
Hi, seems like S0 = 1. is the preferred choice in most tests. So, I am setting S0 as 1.

**Comment by sahmed95** [comment]
This is up for review.

File: dipy/reconst/mapmri.py [hunk 1]
```diff
@@ -108,7 +108,7 @@ def __init__(self,
         >>> data, gtab = dsi_voxels()
         >>> sphere = get_sphere('symmetric724')
         >>> from dipy.sims.voxel import SticksAndBall
-        >>> data, golden_directions = SticksAndBall(gtab, d=0.0015, S0=1, angles=[(0, 0), (90, 0)], fractions=[50, 50], snr=None)
+        >>> data, golden_directions = SticksAndBall(gtab, d=0.0015, S0=100, angles=[(0, 0), (90, 0)], fractions=[50, 50], snr=None)
```
Comment (arokem) [inline dipy/reconst/mapmri.py]: While we're at it, would you mind making this PEP8 compliant, by introducing a line-break? 

It should look like this: 

>>> data, golden_directions = SticksAndBall(gtab, d=0.0015, S0=100,
... angles=[(0, 0), (90, 0)], fractions=[50, 50], snr=None)

Comment (arokem) [inline dipy/reconst/mapmri.py]: Trying again: 

    >>> data, golden_directions = SticksAndBall(gtab, d=0.0015, S0=100,
    ... angles=[(0, 0), (90, 0)], fractions=[50, 50], snr=None)

Comment (arokem) [inline dipy/reconst/mapmri.py]: Yes - like my second attempt (hard to indent and align things in this GH textbox...).

**Comment by arokem** [comment]
It needs 3 dots (`...`) at the beginning of every line in the docstring example. Just like you would get if you ran this in an interactive python session, and put in a line-break there.

**Comment by arokem** [comment]
As in the example I gave in my comment above.

**Comment by arokem** [comment]
You can run the docstring example tests by running
    
    nosetests --with-doctest

**Comment by sahmed95** [comment]
This should do it. I picked up some more pep8's on the way. I hope that's okay.

**Comment by arokem** [comment]
Thanks!

**Pull request merged**